### ISSUE TITLE
Doesn't work with latest `truffle`

### ISSUE THREAD

**Issue by makar-al** [rkalis/truffle-plugin-verify#92]
Fails with `Cannot find module 'project:/contracts/FooBarBaz.sol'` error message. I guess it's because of [this](https://github.com/trufflesuite/truffle/pull/4137) change in `truffle`.

**Comment by rkalis** [comment]
Oof, yep. Good catch. I'll have to update the plugin to work with both ways then. If anyone runs into this issue in the mean time, the current workaround is downgrading to truffle v5.3.13 or lower.

**Comment by jcihocki** [comment]
```
# npx truffle version
Truffle v5.3.12 (core: 5.3.12)
Solidity - 0.8.6 (solc-js)
Node v16.3.0
Web3.js v1.3.6
```

```
# npx truffle run verify MyContract --network mainnet
Verifying MyContract
Cannot find module 'project:/contracts/MyContract.sol'
Require stack:
- /usr/src/zodiac-contract/node_modules/truffle-plugin-verify/verify.js
- /usr/src/zodiac-contract/node_modules/original-require/index.js
- /usr/src/zodiac-contract/node_modules/truffle/build/cli.bundled.js
Failed to verify 1 contract(s): MyContract
```

Are we sure that's the fix?

**Comment by rkalis** [comment]
That's weird. Did you also use the earlier version of Truffle when compiling/migrating?

**Comment by Delgus** [comment]
@rkalis 

truffle-plugin-verify 0.5.10

`
# truffle version
Truffle v5.3.13 (core: 5.3.13)
Solidity - 0.8.0 (solc-js)
Node v10.19.0
Web3.js v1.3.6`

`# truffle run verify APPL@0xEfb1b23b14c989107ac6677Bee0CE271144c5957 --network testnet
Verifying APPL@0xEfb1b23b14c989107ac6677Bee0CE271144c5957
Cannot find module 'project:/contracts/APPL.sol'
Failed to verify 1 contract(s): APPL@0xEfb1b23b14c989107ac6677Bee0CE271144c5957`

**Comment by Delgus** [comment]
I am done deepest downgrade
```
truffle v5.3.1
truffle-plugin-verify 0.5.3
```

It is not work too

**Comment by rkalis** [comment]
Did you also use that version of Truffle when compiling/migrating, or only when verifying?

**Comment by Delgus** [comment]
Yes, i am execute now
```
# truffle compile
# truffle deploy ---network testnet
```
and 
```
# truffle run verify APPL@0xE56038414aeC1a7AfB58CD605D992644cF0Cac3b --network testnet
```

```
Verifying APPL@0xE56038414aeC1a7AfB58CD605D992644cF0Cac3b
Cannot find module 'project:/contracts/APPL.sol'
Failed to verify 1 contract(s): APPL@0xE56038414aeC1a7AfB58CD605D992644cF0Cac3b
```

package.json
``````

**Comment by Delgus** [comment]
@rkalis, i'm sorry
I am manually remove build/contracts and compile again. It's worked!

**Comment by jcihocki** [comment]
> That's weird. Did you also use the earlier version of Truffle when compiling/migrating?

no i didn't, and recompiling all definitely fixed it 🤓 

thanks for responding!

**Comment by rkalis** [comment]
Happy the workaround worked for you guys!

I just released v0.5.11 which should fix the issue for the new version of Truffle as well. If issues persist, please feel free to re-open this issue.
